components/campaigns/AnnoucementHero.js

```diff
@@ -1,31 +1,25 @@
 import { useRouter } from 'next/router'
-import AnnouncementRemainingDays from './AnnouncementRamainingDays'
 import Button from '../buttons/Button'
 
 export default function AnnouncementHero({ className = '', small = false }) {
   const { pathname } = useRouter()
-  const hackathonStartDateTime = '2021-10-01T06:00:00Z'
-  const conferenceStartDateTime = '2021-11-16T06:00:00Z'
-  const hackathonDeadlineDateTime = '2021-10-31T16:00:00Z'
-  const conferenceDeadlineDateTime = '2021-11-18T16:00:00Z'
-  const isHackathonOver = new Date(hackathonDeadlineDateTime) < new Date()
-  const isConferenceOver = new Date(conferenceDeadlineDateTime) < new Date()
 
   if (pathname === '/blog/events2021') return null
-  if (isHackathonOver && isConferenceOver) return null
 
   return (
-    <div className={`mb-12 px-4 ${small ? 'sm:py-6' : 'sm:py-12'} ${className}`}>
-      <h1 className={`text-4xl sm:text-4xl md:text-5xl`}>
-        { !isHackathonOver && <AnnouncementRemainingDays dateTime={hackathonStartDateTime} eventName="Hackathon" /> }
-        { !isConferenceOver && <AnnouncementRemainingDays dateTime={conferenceStartDateTime} eventName="Conference" /> }
-      </h1>
-      <div className="my-2 text-xl text-gray-500">
-        <p className="font-bold text-gray-700">Connect and innovate with the AsyncAPI Community.</p>
-      </div>
-      <div className="mt-8">
-        <Button href="https://conference.asyncapi.com/" target="_blank" text="Learn more" />
+    <div className={`bg-gray-50 border border-gray-200 py-6  ${className} ${small ? 'mb-0' : 'mb-12'}`}>
+      <h2 className="text-xl md:text-3xl font-bold countdown-text-gradient">AsyncAPI Conference (Nov 16-18)</h2>
+      {!small && 
+        <>
+          <p className=" text-gray-700 md:hidden text-xs  my-2 text-center">Connect and innovate with the AsyncAPI Community.</p>
+          <p className=" text-gray-700 hidden md:block max-w-3xl mx-auto  my-2 text-center">The first day of the conference is dedicated entirely for folks that want to contribute to AsyncAPI Initiative. The next two days of the conference are a pure learning experience, community for the community.</p>
+        </>
+      }
+      <div className="mt-8 pb-2 space-x-2">
+        <Button href="https://zoom.us/webinar/register/9416323899487/WN_pv8NDGIoSg2CnF9qsoptWw" target="_blank" text="Register" />
```

Comment:
```suggestion
        <Button href="https://zoom.us/webinar/register/8316333610674/WN_pv8NDGIoSg2CnF9qsoptWw" target="_blank" text="Register" />
```

this is a dedicated link that will help me to see how many people registered directly from the banner

Reply:
cool!